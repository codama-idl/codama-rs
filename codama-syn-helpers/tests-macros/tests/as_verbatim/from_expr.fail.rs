use codama_syn_helpers_test_macros::as_verbatim;

#[as_verbatim("hello")]
pub struct Test;

fn main() {}
