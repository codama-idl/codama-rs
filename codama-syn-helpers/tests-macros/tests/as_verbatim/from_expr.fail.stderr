error: expected verbatim
 --> tests/as_verbatim/from_expr.fail.rs:3:15
  |
3 | #[as_verbatim("hello")]
  |               ^^^^^^^
